python/paddle/fluid/layers/nn.py

```diff
@@ -352,6 +353,38 @@ def fc(input,
     return helper.append_activation(pre_activation)
 
 
+def center_loss(input,
+                labels,
+                num_classes,
+                alpha,
+                param_attr,
+                update_center=True):
```

Comment:
detailed document is missing here!!! It is very important!

Reply:
done